src/commands/up.rs

```diff
@@ -165,13 +174,18 @@ impl UpCommand {
 
         let local_app_dir = app_source.local_app_dir().map(Into::into);
 
-        let run_opts = RunTriggerOpts {
-            locked_app,
-            working_dir,
-            local_app_dir,
-        };
-
-        self.run_trigger(trigger_cmd, Some(run_opts)).await
+        join_all(trigger_cmd.iter().map(|cmd| {
```

Comment:
I believe this should exit as soon as _any_ trigger exits - otherwise the user could end up with only half their application running.

That said, there has been vague talk of a `run` or `once` trigger, which would invoke its guest and then exit.  If that trigger existed, we'd need to be more nuanced about cancelling other triggers in the same application.  That's speculative at the moment though!

Reply:
Changed to try_join_all that seems to give this effect.